implot-sys-bindgen/Cargo.toml

```diff
@@ -4,9 +4,6 @@ version = "0.1.0"
 authors = ["Sandro Merkli"]
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-bindgen = "0.53.1"
+bindgen = "0.55.1"
```

Comment:
Maybe this bump lead to the `i32` -> `u32` change? Anyway for those cases that I checked, the i32 is actually more correct because it mirrors the `int` that the C bindings have. So it's a welcome change in my book.